src/server/main.ts

```diff
@@ -25,17 +28,25 @@ app.post('/register', async (req, res) => {
   if (!userExists) {
     const writeUser = await databaseService.writeUser({ email, username, password });
     console.log(req.body);
-    res.json({ writeUser, status: 'Registration successful!' });
+    res.status(200).json({ writeUser, status: 'Registration successful!' });
   }
   res.json('Username or email already exists');
 });
 
-// TODO
-// post route for login
-// it checks that credentials are valid
-// if they aren't valid, return an error
-// if they are valid, generate a JWT and send it back to the user
-// log them in
+app.post('/login', async (req, res) => {
+  const { username, password } = req.body;
+
+  const isValidUser = await databaseService.verifyUser(username, password);
```

Comment:
also not sure if `verifyUser` is the right word, since this is the database service you are merely checking if the user exists